ground/src/main/java/com/google/android/ground/repository/LocationOfInterestRepository.kt

```diff
@@ -84,6 +87,15 @@ constructor(
         Single.error { NotFoundException("Location of interest not found $locationOfInterest") }
       )
 
+  suspend fun getOfflineLocationOfInterestSuspend(
```

Comment:
LOI is so common in code I think it's ok to abbreviate args, functions, and member vars (i.e. getOfflineLoiSuspend()`).

Reply:
sgtm